### ISSUE TITLE
build error - part of build requires JDK11 (and breaks on 17), while another part requires JDK17 (and breaks on 11)

### ISSUE THREAD

**Issue by mattStorer** [OHSUCMP/ecp-rds#2]
trying to get the ecp-rds system to build without any modifications from its fork from hapi-fhir-jpa-starter, and encountering build issues.

I filed an issue with the hapi-fhir-jpa-starter project, hopefully someone responds.  I'm going to keep banging on it, maybe I'll sort it out before someone gets back to me.

see https://github.com/hapifhir/hapi-fhir-jpaserver-starter/issues/488 for details.

**Comment by mattStorer** [comment]
resolved.  solution is to:
- use JDK 11;
- disable the Maven enforcer plugin in pom.xml via `<enforcer.skip>true</enforcer.skip>`; and
- disable tests on all Maven executions via `-Dmaven.test.skip`

can then run the project through Jetty via `mvn -Dmaven.test.skip jetty:run` and that seems to do the trick

**Comment by mattStorer** [comment]
digging into this a bit more, issue was resolved appropriately by setting Java default to 17 *and* ensuring that Maven references Java 17 as well (it was previously using version 11).  once I got Java 17 overriding 11 in my system path, and by setting JAVA_HOME to reference the Java 17 path (which Maven uses to determine which version of Java to use), I was able to remove the `<enforcer.skip>true</enforcer.skip>` bit in the pom, and run with tests enabled, and it's now working as expected.  woot.
